pkg/cmd/server/server.go

```diff
@@ -776,30 +775,15 @@ func (s *server) runControllers(defaultVolumeSnapshotLocations map[string]string
 		}
 	}
 
-	serverStatusRequestControllerRunInfo := func() controllerRunInfo {
-		serverStatusRequestController := controller.NewServerStatusRequestController(
-			s.logger,
-			s.veleroClient.VeleroV1(),
-			s.sharedInformerFactory.Velero().V1().ServerStatusRequests(),
-			s.pluginRegistry,
-		)
-
-		return controllerRunInfo{
-			controller: serverStatusRequestController,
-			numWorkers: defaultControllerWorkers,
-		}
-	}
-
 	enabledControllers := map[string]func() controllerRunInfo{
-		BackupSyncControllerKey:          backupSyncControllerRunInfo,
-		BackupControllerKey:              backupControllerRunInfo,
-		ScheduleControllerKey:            scheduleControllerRunInfo,
-		GcControllerKey:                  gcControllerRunInfo,
-		BackupDeletionControllerKey:      deletionControllerRunInfo,
-		RestoreControllerKey:             restoreControllerRunInfo,
-		ResticRepoControllerKey:          resticRepoControllerRunInfo,
-		DownloadRequestControllerKey:     downloadrequestControllerRunInfo,
-		ServerStatusRequestControllerKey: serverStatusRequestControllerRunInfo,
```

Comment:
This wasn't a big deal for the BSL controller, but it is here - this is a breaking change. With this change, users can no longer specify whether or not the controller can be enabled or not, unless I'm missing something.

The problem will be amplified as we move more controllers, so we'll have to figure out how to retain this functionality.

Reply:
Ah yes, I had missed that, good catch. Fixed it: https://github.com/vmware-tanzu/velero/pull/2838/files#diff-cd1a009fa409db8496ea35b53f9776d2R790.

Reply:
Awesome, thanks!